Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4_Interstage.cfg

```diff
@@ -16,7 +16,7 @@ MODEL
 	TechRequired = specializedConstruction
 	entryCost = 8000
 	cost = 2530
-	category = Structural
+	category = Development -- I think you have a good idea with the first paragraph, and it definetly has a place, but its presence at the start is a bit jarring. Perhaps rewrite it into the first or move the thesis up. Either way, the first and second paragraphs together make a good opening with a great hook
```

Comment:
Oh, crap, that's embarrassing. That'll teach me to try to do two things at once.